ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/ConnectorType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.client;
+
+public interface ConnectorType {
+  enum Type {
+    SOURCE,
+    SINK,
+    /**
+     * Denotes an unknown connector type. This is used when there were errors in connector creation.
```

Comment:
Is this the only reason the connector type might be `UNKNOWN` or are there others as well, for example, when listing existing connectors? For some reason I thought there were other possible reasons for an `UNKNOWN` type as well but I could be wrong.

Reply:
I couldn't find any other references to `UNKNOWN`, but there could be a reason on the Connect side of things that I'm missing. Updated docs to be more accurate